core/test/bitcoin/bitcoin_utxo_picket_tests.cpp

```diff
@@ -185,3 +184,58 @@ TEST(OptimizeSize, ApproximationShouldTookEnough) {
     if (buddy->changeAmount.toInt64() != 0)
         EXPECT_GE(buddy->changeAmount.toInt64(), inputSizeInBytes * feesPerByte);
 }
+
+void feeIsEnoughFor(const std::string address, const int64_t targetOutputSizeInBytes) {
+    const api::Currency currency             = currencies::BITCOIN_TESTNET;
+    const int64_t feesPerByte                = 1;
```

Comment:
what if feesPerByte = 1.2 ?

Reply:
I've tried a range of values of `feesPerByte` between 1 and 10000000 (see updated test). It works as expected.